lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +84,43 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        with open(self.path_to_config) as file:
+            file = json.load(file)
```

Comment:
do not overwrite same variable. name it differently

Reply:
done